src/main/java/frc/robot/subsystems/DriveBase.java

```diff
@@ -58,6 +58,9 @@ private DriveBase() {
 
     }
 
+    /**
+     * Initialize the current DriveBase instance
+     */
     public static void init() {
```

Comment:
Again with the standardize on the initialization method. See the IntakeSubsystem for an example

Reply:
I'll deal with this in a separate PR as this is unrelated to the current one.